apps/backend/src/resolvers/video-clip.resolver.ts

```diff
@@ -29,14 +29,16 @@ export class VideoClipResolver {
     @Arg('limit', () => Int, { nullable: true, defaultValue: 12 })
     limit?: number,
     @Arg('sortBy', () => String, { nullable: true }) sortBy?: string,
-    @Arg('filterShow', () => String, { nullable: true }) filterShow?: string
+    @Arg('filterShow', () => String, { nullable: true }) filterShow?: string,
+    @Arg('filterCharacter', () => String, { nullable: true }) filterCharacter?: string
```

Comment:
Create a new `VideoClipFilter` input type that would include the `searchQuery`, `filterShow`, and `filterCharacter` instead of passing them individually. This type would be passed to the `videoClips`, `availableShows` and `availableCharacters` queries.

Also update the frontend with the updated queries.